apps/resources-wcm/src/main/webapp/WEB-INF/gatein-resources.xml

```diff
@@ -326,24 +326,6 @@
     </script>
   </module>
 
-  <module>
-    <name>document-preview</name>
```

Comment:
Please check that this JS module is not used in GTMPL files

Reply:
after searching , there are 2 gtmpl vues in ecm-wcm-extension using a module named document-preview but this module is configured to use only the right document-preview file which is document-preview.js and not DocumentPreview.js

Reply:
Ok thanks